python/scripts/pyshell

```diff
@@ -0,0 +1,197 @@
+COMMAND_MODE='SHELL_MODE' 
+RUNNING_MODE='HADOOP'
+EXECUTABLE_FILE=false
+while getopts ":s:l:k:t:m:" o; do
+    case "${o}" in
+        s)
+            s=${OPTARG}
+            COMMAND_MODE="SHELL_MODE"
+            ;;
+        l)
+            EXECUTABLE_FILE=${OPTARG}
+            COMMAND_MODE="LAUNCH_MODE"
+            ;;
+        k)  
+            APP_NAME=${OPTARG}
+            COMMAND_MODE="KILL_MODE"
+            ;;
+        m)  
+            RUNNING_MODE=${OPTARG}
+            ;;
+        t) 
+            usage
+            echo "NOT IMPLEMENTTED YET"
+            ;;
+        *)
+            usage
+            ;;
+    esac
+done
+echoerr() { echo "$@" 1>&2; }
+if [ -e "pyapex" ]
+then 
+  rm pyapex
+fi
+
+real_dir() {
+  SOURCE="${1:-${BASH_SOURCE[0]}}"
+  while [ -h "$SOURCE" ]; do # resolve $SOURCE until the file is no longer a symlink
+    SOURCE_DIR="$( cd -P "$( dirname "$SOURCE" )" && pwd )"
+    SOURCE="$(readlink "$SOURCE")"
+    [[ $SOURCE != /* ]] && SOURCE="$SOURCE_DIR/$SOURCE" # if $SOURCE was a relative symlink, we need to resolve it relative to the path where the symlink file was located
+  done
+  SOURCE_DIR="$( cd -P "$( dirname "$SOURCE" )" && pwd )"
+  echo $SOURCE_DIR
+}
+script_dir=$(real_dir "${BASH_SOURCE[0]}")
+# Create missing clirc file for current user
+if [ ! -f "${HOME}/.dt/clirc" ]; then
+  mkdir -p "${HOME}/.dt"
+  cat >${HOME}/.dt/clirc <<EOF
+# User editable apex settings
+EOF
+fi
+# Load DataTorrent environment settings
+for conf_dir in "${script_dir}/../conf" "$HOME/.dt"; do
+    [[ -f "${conf_dir}/dt-env.sh" ]] && . "${conf_dir}/dt-env.sh"
+done
+
+# In development mode, if configuration files are not found, locate DT_HADOOP manually
+if [ -z "${DT_HADOOP}" ]; then
+  HADOOP_SEARCH_PATH="${HADOOP_PREFIX}/bin:${HADOOP_HOME}/bin:${PATH}:/usr/local/bin:/bin:/usr/bin:/usr/local/sbin:/usr/sbin:/sbin:."
+  export DT_HADOOP=`PATH=${HADOOP_SEARCH_PATH} && command -v hadoop 2>/dev/null`
+fi
+
+if [ "$DT_CLIENT_OPTS" = "" ]; then
+#  DT_CLIENT_OPTS="-Xmx1024m -XX:MaxPermSize=512m -XX:+UseConcMarkSweepGC -XX:+CMSClassUnloadingEnabled"
+  DT_CLIENT_OPTS="-Xmx1024m -XX:+UseConcMarkSweepGC -XX:+CMSClassUnloadingEnabled"
+fi
+
+export HADOOP_CLIENT_OPTS="$DT_CLIENT_OPTS"
+
+PYTHON_BUILD="$( dirname "$0" )/../dist"
+BUILD_DIR="$( dirname "$0" )/../target"
+PYAPEX_HOME="`pwd`/../apex-python"
+
+if [[ (-z "$DT_HADOOP" ) || ( "$RUNNING_MODE" == "LOCAL" ) ]]; then
+  echo "Development Mode without Hadoop Installation: Using jars from mvn generated path"
+  MVN_GENERATED_PATH="$BUILD_DIR/mvn-generated-runtime-classpath"
+else
+  echo "Development Mode with Hadoop Installation:  Using jars to be provided externally "
+  MVN_GENERATED_PATH="$BUILD_DIR/mvn-generated-runtime-classpath-no-hadoop"
+fi
+if [ -f "$MVN_GENERATED_PATH" ]; then
+  # development launch mode
+  DT_CORE_JAR="$BUILD_DIR/malhar-python-3.8.0-SNAPSHOT.jar"
```

Comment:
Check how I can include current artifact in the classpath files like MVN_GENERATED_PATH